guide/src/building_and_distribution.md

```diff
@@ -58,6 +58,17 @@ pyo3 = { version = "...", features = ["abi3"]}
 
 3. Ensure that the `.whl` is correctly marked as `abi3`. For projects using `setuptools`, this is accomplished by passing `--py-limited-api=cp3x` (where `x` is the minimum Python version supported by the wheel, e.g. `--py-limited-api=cp35` for Python 3.5) to `setup.py bdist_wheel`.
 
+### Minimum Python version for `abi3`
+
+We provide `abi3-py36`/`abi3-py37`/... features to set the minimum required Python version for abi3 wheel.
+E.g., if you set `abi3-py36` feature, you can build `cp36-abi3-manylinux2020_x86_64.whl` using Python 3.8.
+
```

Comment:
Let's maybe add a new paragraph here explaining what happens if you set multiple:

```suggestion
If you set more that one of these api version feature flags the highest version always wins. For example, with both `abi3-py36` and `abi3-py38` set, PyO3 would build a wheel which supports Python 3.8 and up.
```